content/docs/services/cdn-route.md

```diff
@@ -8,30 +8,40 @@ description: "Custom routes with HTTPS certificates using Amazon CloudFront"
 status: "Production Ready"
 ---
 
-Once your application is ready to move to production, you'll want a custom domain instead of the default cloud.gov one. The CDN Route service provides three key elements to support production applications: a custom domain, Content Distribution Network (CDN) caching, and TLS certificates with auto renewal.
+When you want a custom domain instead of the default `*.app.cloud.gov` domain, use the CDN service. It provides three key elements to support production applications: custom domain use, Content Distribution Network (CDN) caching (using [AWS CloudFront](https://aws.amazon.com/cloudfront/)), and free TLS certificates with auto renewal.
 
 ## Plans
 
 Plan Name | Description | Price
 --------- | ----------- | -----
-`cdn-route` | A CDN distribution with custom URL and free TLS certificate with auto renewal | Free
+`cdn-route` | A CDN distribution with custom domain and free TLS certificate with auto renewal | Free
 
 ## Options
 
 Name | Required | Description | Default
 --- | --- | --- | ---
-`domain` | <i class="fa fa-check" aria-hidden="true"></i> | Your custom domain (or domains separated by commas) |
-`origin` | <i class="fa fa-check" aria-hidden="true"></i> | The origin root URL of the application |
-`path` |  | The path for the application within the main URL supplied | `""`
-`insecure_origin` |  | Read the application over HTTP instead of HTTPS | `false`
+`domain` | *Required* | Your custom domain (or domains separated by commas) |
+`origin` | *Optional* | *Don't put this in your command for cloud.gov tenant applications.* For services/applications that are not cloud.gov tenant applications ([more info](#external-services-and-applications)): the origin root URL of the application |
+`path` | *Optional* | The path for the application within the main domain supplied | `""`
+`insecure_origin` | *Optional* | Read the application over HTTP instead of HTTPS | `false`
 
-## How to create an instance
+## How to create an instance of this service
 
-To create a service instance, run the following command:
+*Use these instructions for cloud.gov tenant applications. If you're creating a custom domain for something else (such as a public S3 bucket), see [external services and applications](#external-services-and-applications).*
+
+Note: CDN service instances cannot be updated or deleted until they have been successfully configured. Until this issue is resolved, please double-check all commands related to provisioning and updating instances.
```

Comment:
What do you mean by "CDN service instances cannot be updated or deleted until they have been successfully configured"? How would you manage to successfully configure this if you make a mistake?